idrisadetunmbi/github-android-developers
# #155965911 Configure static analysis tools and pre-commit hook

#### What does this PR do?
Configures static analysis tools including findbugs, checkstyle, pmd and lint.

#### Description of Task to be completed?
Add a unit test, clean build,  java code style checks & lint checks  pre-commit hook

#### How should this be manually tested?
Clone the repo in Android Studio. Run `./gradlew check` or other gradle commands that lists `check` as dependency and observe the SCA run

#### Any background context you want to provide?
The pre-commit hook is configured per project and is not available in the repo since it resides in the `.git` folder locally

#### What are the relevant pivotal tracker stories?
#155965911

#### Screenshots (if appropriate)
N/A

#### Questions:
